src/main/java/io/fabric8/elasticsearch/plugin/kibana/KibanaSeed.java

```diff
@@ -269,9 +269,14 @@ private void buildUserAlias(List<String> projects, String projectPrefix, String
             if (projects.isEmpty()) {
                 return;
             }
+            Set<String> aliasedIndicies = pluginClient.getIndicesForAlias(alias);
+            aliasedIndicies = new IndexUtil().replaceDateSuffix("*", aliasedIndicies);
             Map<String, String> patternAlias = new HashMap<>(projects.size());
             for (String project : projects) {
-                patternAlias.put(getIndexPattern(project, projectPrefix), alias);
+                String indexPattern = getIndexPattern(project, projectPrefix);
+                if(!aliasedIndicies.contains(indexPattern)) {
```

Comment:
do the `indexPattern, alias` pairs in `aliasedIndices` get added to `patternAlias` somewhere else, or are they not needed any more?

Reply:
They are not needed any longer and are discarded.